janusgraph-core/src/main/java/org/janusgraph/graphdb/schema/EdgeLabelDefinition.java

```diff
@@ -20,7 +20,10 @@
 
 /**
  * @author Matthias Broecheler (me@matthiasb.com)
+ *
+ * @deprecated part of the management revamp in JG, see https://github.com/JanusGraph/janusgraph/projects/3.
```

Comment:
I am just trying to understand: how is this related to the management revamp? I can see even in v0.5 branch, these classes are only used in tests.

Reply:
You could you use the explore schema. I don't think anyone is using it.

Reply:
Can you elaborate on "explore schema"? Never heard of it.

Reply:
You could you use the schema provider to list all vertices, edges or properties.

Reply:
Got it.